app/uk/gov/hmrc/agentsubscriptionfrontend/auth/MtdSubscriberRegime.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentsubscriptionfrontend.auth
+
+import uk.gov.hmrc.play.frontend.auth
+import uk.gov.hmrc.play.frontend.auth.TaxRegime
+import uk.gov.hmrc.play.frontend.auth.connectors.domain.Accounts
+
+object MtdSubscriberRegime extends TaxRegime {
+  override def isAuthorised(accounts: Accounts) = accounts.sa.isDefined
```

Comment:
We're expecting the agent to use a new GG cred so they won't have an IR-SA enrolment (which is what accounts.sa is mapped from).

I think we probably want isAuthorised to always return true (or something else that provides equivalent behaviour, i.e. you have to be logged in but we don't perform any further checks)